cmplxs/cmplxs.go

```diff
@@ -564,9 +565,7 @@ func Real(dst []float64, src []complex128) []float64 {
 
 // Reverse reverses the order of elements in the slice.
```

Comment:
Suggest adding deprecation notice pointing to `slices.Reverse`.
```
// Deprecated: This function simply calls [slices.Reverse].
```

Reply:
Makes sense to me. [Done](https://github.com/gonum/gonum/pull/1941/commits/9ea21b5adf77baec5177d9da143b064cd6c77105).

Reply:
I'll pore over my other PRs and figure out which ones could do with deprecation notices like this.

Reply:
Make this 
```
// Reverse reverses the order of elements in the slice.
//
// Deprecated: This function simply calls [slices.Reverse].
```

Reply:
@kortschak Done! :+1: